pkg/worker/worker.go

```diff
@@ -42,6 +42,11 @@ func (w *Worker) Run(eventChannel *channels.RingChannel, stopChannel chan struct
 					glog.V(5).Infof("Received event: %s", jsonEvent)
 				}
 
+				if !w.ShouldProcess(event){
+					glog.V(5).Infof("Skipping event")
```

Comment:
A very very helpful message here would be to indicate why we are skipping the event.